Fix typing indicator showing on the middle of an empty message list (#1545)
GetStream/stream-chat-swiftui: CHANGELOG.md

### 🐞 Fixed
- Pass the channel's team when starting a direct message with a member [#1541](https://github.com/GetStream/stream-chat-swiftui/pull/1541)
- Fix Markdown links with URL-shaped display text opening the wrong destination, e.g. `[https://text-link.com](https://real-link.com)` [#1543](https://github.com/GetStream/stream-chat-swiftui/pull/1543)

> - Show the typing indicator at the bottom of the message list instead of the middle of the screen when a channel has no messages [#1545](https://github.com/GetStream/stream-chat-swiftui/pull/1545)

Reviewer: A bit simpler this way, and no need to mention top or bottom aligned.
```suggestion
- Fix showing the typing indicator in the middle of the screen when a channel has no messages [#1545](https://github.com/GetStream/stream-chat-swiftui/pull/1545)
```
